packages/veritone-react-common/src/components/ProgramInfo/Affiliates/index.js

```diff
@@ -119,20 +114,20 @@ export default class Affiliates extends React.Component {
         delete newAffiliate.schedule.weekly[selectedDay];
       }
     }
-    const affiliateById = {
-      ...this.props.selectedAffiliateById
+    const newAffiliateById = {
+      ...this.props.affiliateById
     };
-    affiliateById[newAffiliate.id] = newAffiliate;
-    this.props.onAffiliatesChange(affiliateById);
+    newAffiliateById[newAffiliate.id] = newAffiliate;
+    this.props.onAffiliatesChange(newAffiliateById);
```

Comment:
same as above

this.props.onAffiliatesChange({
...this.props.affiliateById,
[newAffiliate.id]: newAffiliate
})